compatibility.md

```diff
@@ -43,10 +43,11 @@ version.
 | `fastify` | 2.0.0 | 4.27.0 | 8.5.0 |
 | `generic-pool` | 2.4.0 | 3.9.0 | 0.9.0 |
 | `ioredis` | 3.0.0 | 5.4.1 | 1.26.2 |
+| `kafkajs` | 2.0.0 | 2.2.4 | 11.19.0 |
```

Comment:
This PR was triggered by adding this library to the versioned tests. We are not ready for this one yet.